### ISSUE TITLE
Pass array in mariadb query

### ISSUE THREAD

**Issue by sakuriki** [MariaDB/mariadb-connector-nodejs#64]
Hello,
I have a simple query like this, 
``db.query("DELETE `tb_a`, `tb_b` FROM `tb_a` LEFT JOIN `tb_b` ON `tb_b`.a_key = `tb_a`.key WHERE `tb_a`.key IN ('"+key.join("','")+"')")``
``//key = ['abc','gfd']``
It work well but I want to prevent sql inject so I intent to use placeholder like this
``db.query("DELETE `tb_a`, `tb_b` FROM `tb_a` LEFT JOIN `tb_b` ON `tb_b`.a_key = `tb_a`.key WHERE `tb_a`.key IN (?)", [key])``
But it doesn't seem to work
``{ affectedRows: 0, insertId: 0, warningStatus: 0 }``
Am I doing this right?

**Comment by knoxcard** [comment]
```
db.query("DELETE `tb_a`, `tb_b` FROM `tb_a` LEFT JOIN `tb_b` ON `tb_b`.a_key = `tb_a`.key WHERE `tb_a`.key IN (?)", [key])
```

**Comment by knoxcard** [comment]
Appears to be fixed ... https://github.com/MariaDB/mariadb-connector-nodejs/commit/2d06e2f3bd9c152e5886ddc5c7519d0d717345ec

@sakuriki - can you pull the latest and try again?

Close ticket?

**Comment by rusher** [comment]
yes, commit done on `develop` branch.
This will be released in 2.1.0 (end of this week). 

This wasn't done initially because this would have been different for a future evolution (using binary protocol). !There is finally some solution for this futur evolution. So no reason to handle that use case.
